src/app/data/oasis-account-names.ts

```diff
@@ -34,6 +34,10 @@ const dataSources: Record<Network, Partial<Record<Layer, string>>> = {
       'https://raw.githubusercontent.com/oasisprotocol/nexus/main/named-addresses/testnet_emerald.json',
     [Layer.sapphire]:
       'https://raw.githubusercontent.com/oasisprotocol/nexus/main/named-addresses/testnet_sapphire.json',
+    [Layer.pontusxdev]:
+      'https://raw.githubusercontent.com/oasisprotocol/nexus/main/named-addresses/testnet_pontusxdev.json',
```

Comment:
(can we just copy in pontusx's other account names too, to simplify our code?)

Reply:
Their list changes much more often than hours.

Plus, it's easy to keep the source info correctly if the sources are separate... but maybe we can rethink this when we make this a proper nexus API.